components/counter/src/scripts/version.mjs

```diff
@@ -0,0 +1,3 @@
+import { writeDepVersionFile } from '../../../../scripts/formVersionWriter.js';
+
+writeDepVersionFile('@aurodesignsystem/auro-icon');
```

Comment:
You'll need to add `await` here:

```
await writeDepVersionFile('@aurodesignsystem/auro-icon');
```

Reply:
eslint does not like the top level await. I added it anyways since all the others have it.